package.json

```diff
@@ -8,7 +8,11 @@
     "build": "tsc && webpack",
     "test": "vitest",
     "test:ci": "vitest --reporter=junit --outputFile=junit.xml",
-    "coverage": "vitest run --coverage"
+    "coverage": "vitest run --coverage",
+    "lint": "eslint .",
+    "lint:ci": "eslint --output-file eslint-report.json --format json .; exit 0",
```

Comment:
Won't `exit 0` just make this task never fail?
I would think we'd want the pipeline to fail if linting fails

Reply:
Good question! I just blindly copy-pasted it from our ingest pipeline

Reply:
I think that might have been a work-around in the past to not fail the reporting step. But that is taken care of now with custom conditions. Will update.